src/pineko/evolve.py

```diff
@@ -46,6 +49,52 @@ def sv_scheme(tcard):
     return modsv
 
 
+def get_grid_convolution_type(kv):
+    """Retrieve the ekos convolution type.
+
+    Parameters
+    ----------
+    kv: dict
+        pineappl grid metadata
+    """
+    # TODO: This should probably changed in the future to use the Grid::convolutions
+    if "convolution_type_1" in kv:
+        conv_type_1 = kv["convolution_type_1"]
+    # TODO: polarized is now deprecated, needed for compatibility
+    elif "polarized" in kv and kv["polarized"] == "True":
+        conv_type_1 = "PolPDF"
+    else:
+        conv_type_1 = "UnpolPDF"
+
+    # TODO: initial_state_2 is now deprecated, needed for compatibility
```

Comment:
I think yadism writes 2212 also for other targets: 
https://github.com/NNPDF/yadism/blob/7d9e2db488622761eda72282f73317602bc8a39a/src/yadbox/export.py#L88

Reply:
I'm not sure that's good ^^U

I know NNLOJET e.g., writes the antiproton as -2212 and to get the tevatron fktable it is necessary to conjugate them first though.

Reply:
> I think yadism writes 2212 also for other targets: https://github.com/NNPDF/yadism/blob/7d9e2db488622761eda72282f73317602bc8a39a/src/yadbox/export.py#L88

this is because yadism _rescales_ PDF to proton, i.e. if you use "nuclear corrections" in yadism, we do the "LO" thing: i.e. we [change u and d around](https://yadism.readthedocs.io/en/latest/theory/misc.html#isospin).

For an arbitrary grid, which is the case here in pineko, we can not assume anything.

Reply:
I think this way is correct, if you wish remove the mention to the proton.